radio/src/storage/yaml/yaml_datastructs_nv14.cpp

```diff
@@ -247,9 +247,9 @@ const struct YamlIdStr enum_PotsWarnMode[] = {
   {  0, NULL  }
 };
 const struct YamlIdStr enum_ModelOverridableEnable[] = {
-  {  OVERRIDE_VALUE_GLOBAL, "GLOBAL"  },
-  {  OVERRIDE_VALUE_OFF, "OFF"  },
-  {  OVERRIDE_VALUE_ON, "ON"  },
+  {  OVERRIDE_VALUE_GLOBAL, "VALUE_GLOBAL"  },
+  {  OVERRIDE_VALUE_OFF, "VALUE_OFF"  },
+  {  OVERRIDE_VALUE_ON, "VALUE_ON"  },
```

Comment:
Why the need to add VALUE_ prefix for yaml?

Reply:
Unsure, that got genrated from the yaml generator